### DISCUSSION TITLE
Enhanced Bottom Bar Tab Navigation for Improved Usability

### DISCUSSION THREAD

**Discussion by cryptoluks** [LeoKlaus/Paperparrot#156] [Feature Requests]
Hey Leo,

First off, big thanks for making Paperparrot. It's been super helpful!

It would be cool if clicking the bottom bar buttons took you back to their main menus.

For instance, if I'm checking out the details of a document and I tap on the 'Documents' tab again, I think it'd be handy to jump back to the document overview. Same goes for other sections like Search or Settings - tapping the same tab could bring you back to the top of that page.

I've seen this in some of Apple's apps, like Files, and it really feels intuitive. What do you think, Leo? Could be a nice touch to make navigating around a bit smoother.

Thanks for all your hard work on this!

**Comment by LeoKlaus** [comment]
Hey cryptoluks,

first off, thank you for reaching out and your feedback!
I've actually already tried implementing this, but SwiftUI makes it harder than it should be (especially while keeping the current iOS 15 target).

It's been a while though and this is something I personally would enjoy too, so I'll take another look at it.

**Comment by LeoKlaus** [comment]
Implemented.
